great_expectations/data_context/migrator/cloud_migrator.py

```diff
@@ -96,34 +96,34 @@ def migrate(
         cls,
         context: BaseDataContext,
         test_migrate: bool,
-        ge_cloud_base_url: Optional[str] = None,
-        ge_cloud_access_token: Optional[str] = None,
-        ge_cloud_organization_id: Optional[str] = None,
+        cloud_base_url: Optional[str] = None,
+        cloud_access_token: Optional[str] = None,
+        cloud_organization_id: Optional[str] = None,
```

Comment:
Figured this could just be changed instead of deprecating due to lack of usage - thoughts?

Reply:
Agree